truss/remote/baseten/service.py

```diff
@@ -0,0 +1,39 @@
+from typing import Dict, Optional
+
+from truss.remote.baseten.auth import AuthService
+from truss.remote.truss_remote import TrussService
+from truss.truss_handle import TrussHandle
+
+
+class BasetenService(TrussService):
+    def __init__(
+        self,
+        model_id: str,
+        model_version_id: str,
+        is_draft: bool,
+        api_key: str,
+        service_url: str,
+        truss_handle: Optional[TrussHandle] = None,
+    ):
+
+        super().__init__(is_draft=is_draft, service_url=service_url)
+        self._model_id = model_id
+        self._model_version_id = model_version_id
+        self._auth_service = AuthService(api_key=api_key)
+        self._truss_handle = truss_handle
+
+    @property
+    def is_live(self) -> bool:
```

Comment:
let's add https://docs.python.org/3/library/exceptions.html#NotImplementedError to both this and readiness probes for now. No need to block on this and return an invalid value is not great behavior.